### ISSUE TITLE
Can't open any anime episode list. 

### ISSUE THREAD

**Issue by roworit** [DxCx/plugin.video.9anime#326]
Does anyone else have this problem?
Like in the title I can't open any anime, I can browse the various genre or see the latest updated Anime but I can't open the anime itself, the site works though I just checked.

**Comment by JulGer** [comment]
Same here.

**Comment by Cayz181299** [comment]
Yes same I can't open any amine to watch only to browse

**Comment by JaeFr0st** [comment]
I’m having the same issue.

**Comment by JaeFr0st** [comment]
I’m having the same issue. I’m getting the 9Anime Error check the log info for more information.

**Comment by z89x23** [comment]
Same with JaeFr0st

**Comment by JustAguyWhoLikesAnime** [comment]
can confirm tried watching a ep of something it did not work but when I tried on the site it worked fine

**Comment by RedNinjaX** [comment]
Yeah I see what's happening now. This is...bad. Really bad

It would seem that now 9anime dynamically inserts the servers into the page instead of keeping it in the static html. Not only does this prevent scraping the pages for the episode IDs (which are needed to get the embed links and therefore the video links) which are needed for the plugin but also prevents being able to choose an episode with adblock enabled. On top of that the RapidVideo server seems to be disappearing from a lot of pages, but that could be just temporary.

I'll probably take a look at it when I get home from work but I'll admit they just made it a whole lot difficult for developers of any 9a plugin (with the exception of browser extensions)

**Comment by JustAguyWhoLikesAnime** [comment]
@RedNinjaX  so there may not be a way to make it work anymore?

**Comment by RedNinjaX** [comment]
@JustAguyWhoLikesAnime meh. I think I have an idea of how to fix this but I'm not too sure so I'll find out when I get home

**Comment by JustAguyWhoLikesAnime** [comment]
@RedNinjaX I wish you luck man

**Comment by RedNinjaX** [comment]
@SwagOtaku if it has something to do with all.js I think we maybe of a similar solution

Anyway sure. Give me your Discord tag and I'll add you to the group chat

**Comment by SwagOtaku** [comment]
@RedNinjaX discod tag is swagkush

**Comment by RedNinjaX** [comment]
@SwagOtaku swagkush#{number_here}

**Comment by SwagOtaku** [comment]
@RedNinjaX whoops sorry 
swagkush#1026

**Comment by RedNinjaX** [comment]
@SwagOtaku sent request

**Comment by popsickle123456** [comment]
Any luck on this?

**Comment by RedNinjaX** [comment]
Yeah, thanks to SwagOtaku. It will get fixed soon

**Comment by HatHeadNinja** [comment]
9anime could've made money by implementing IP pairing, like openload, but instead choose to waste their time playing this game, which only costs them. They even broke their own site attempting to defeat scraping.

**Comment by Jkrise** [comment]
You know, if they provide a subscription service, kind of like crunchy roll, but just not down ask the time; I would be willing to pay for that, but instead they waste their time with this b.s.

**Comment by Jkrise** [comment]
By the way, I volunteer as Ginnie pig if you need some testers :D

**Comment by DxCx** [comment]
yeah 9anime changed thier listing technique..
i'll look into it more deeply in the weekend and probably release a fix.

**Comment by z89x23** [comment]
Thank you all for your hard work..

**Comment by Jkrise** [comment]
Yes, thank you

**Comment by Shahav** [comment]
Yes me too.... Everytime i try to watch any episodes i get an error message
to check log; so annoying & irritating but no matter what i do it to
configure the addon or anything else it doesn't help!!!  I hope we can get
this fixed cause i love this addon & have used it for years.  P.S. i am
using my firetv stick right now but tried on my amdroid tv box and no luck
either.

Sherry

**Comment by ruegnueg** [comment]
kodi on firetv Stick? thats def not fun 😉 tried fireanime? libreelec ftw.... by the way im sure it will be fixed soon by the great Team here.

**Comment by ruegnueg** [comment]
ah its time for webvideocaster on my Phone again 😂

**Comment by JaeFr0st** [comment]
I love 9Anime but in my opinion watchcartoononline is way better and way faster in terms of performance. It’s not as nice looking as 9Anime but it does what it needs to do. It loads super fast and it doesn’t search for sources and then you wait and select. You just hit play and select between 720 and SD and that’s it. Takes less than 10 seconds before you can enjoy your shows. 
![c8006d2e-3aa8-4393-9fcf-27ff7b0cac92](https://user-images.githubusercontent.com/35748268/43968787-245fd9f8-9c96-11e8-8fa9-c287fb4532f8.jpeg)

**Comment by Cayz181299** [comment]
Where did you download that watch cartoons one from

**Comment by JaeFr0st** [comment]
@Cayz181299 from here 
![11e8563d-9fcb-4f24-8dc5-6686e93bd36c](https://user-images.githubusercontent.com/35748268/43968968-b817ec9e-9c96-11e8-808f-5d269be1fa7f.jpeg)

**Comment by JaeFr0st** [comment]
@Cayz181299 it’s currently the best anime fork on Kodi. I love it. It’s fast as fuck.

**Comment by JaeFr0st** [comment]
@Cayz181299 no problem. I just hope they fix 9Anime and I wish someone can find a way make it fast like watchcartoon.

**Comment by Cayz181299** [comment]
@JaeFr0st yeahh, it feels like the 9anime guys are purposely doing stuff to the website to break the 9anime addon

**Comment by JaeFr0st** [comment]
😂

**Comment by JulGer** [comment]
You guys do know, that the add-on is working again? Btw. Watchnixtoons is my go-to add-on too.

**Comment by JaeFr0st** [comment]
Wow I did not even know that 🤣 haha. I was so into the watchnixtoons I didn’t even catch that 🤣 thanks for letting me know.

**Comment by dreamkenya** [comment]
Still doesn't work for me yet.

**Comment by Cayz181299** [comment]
@dreamkenya update the add on through github browser on kodi

**Comment by dreamkenya** [comment]
@cayz181299 thank you that worked.

**Comment by ruegnueg** [comment]
it is only subbed on watchnixtoons? that would be shit.

**Comment by JaeFr0st** [comment]
@ruegnueg no. It has both subbed and dubbed.

**Comment by ruegnueg** [comment]
So tried watchnix, its really fast, but only 720p is a downer 😉 but great for times when 9anime addon has temp probs.

**Comment by ruegnueg** [comment]
but no watchlist or anything.... nah.

**Comment by ruegnueg** [comment]
So you guys know animedroid for Phone and Tablet?

**Comment by z89x23** [comment]
Thank you so much i will try it laters

**Comment by Cayz181299** [comment]
Thank you to the wonderful people who work hard to keep this amazing Add-on alive!

**Comment by ahardy10** [comment]
So I didn't have GitHub browser. I installed it on kkodi so I could get the new cypher 9 anime update so it will start working again. I searched up DxCx in the github search. and installed the 4 files that came up at which point my kodi restarted. My cypher 9 anime still isn't working can somebody help please? I'm kind of noob aha.
